examples/librispeech/librispeech.ipynb

```diff
@@ -119,20 +104,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [],
    "source": [
     "use_data_augmentation = False\n",
-    "augmenter = WavAugmenter.create_predefined('pitch_reverb_tdrop', sampling_rate=16000) if use_data_augmentation else None\n",
-    "# It seems when spawning multiple Python subprocesses with the same sox handle it raises \"std::runtime_error: Couldn't close file\"\n",
-    "# The issue seems to happen only in a Jupyter notebook on Mac OS X, hence the work around below.\n",
-    "if use_data_augmentation:\n",
-    "    num_jobs = 1\n",
-    "else:\n",
-    "    num_jobs = os.cpu_count()\n",
-    "    torch.set_num_threads(1)\n",
-    "    torch.set_num_interop_threads(1)"
+    "augment_fn = SoxEffectTransform(effects=[\n",
+    "   ['reverb', 50, 50, RandomValue(0, 100)],\n",
+    "   ['speed', RandomValue(0.9, 1.1)],\n",
```

Comment:
I strongly advise not to use `speed` + `rate` this way because it change the audio array length, but doesn't update the supervisions to match it. The preferred way is to use `cut_set.perturb_speed(0.9)`, and if you want to combine raw and augmented versions, `cut_set = cut_set + cut_set.perturb_speed(0.9) + cut_set.perturb_speed(1.1)`